scripts/pi-hole/php/password.php

```diff
@@ -35,8 +39,23 @@
     // Test if password is set
     if(strlen($pwhash) > 0)
     {
+        // Check for and authorize from persistent cookie 
+        if (isset($_COOKIE["persistentlogin"]))
+        {
+            if ($pwhash = $_COOKIE["persistentlogin"])
+            { $auth = true;
+		// Refresh cookie with new expiry
+                setcookie('persistentlogin', $pwhash, time()+60*60*24*7);
```

Comment:
There's still a mixture of spaces and tabs here, and there should not be anything besides the brace on line 46.

Reply:
done